From fea1e1173637a9267d7afa735a4875b3b39f35f0 Mon Sep 17 00:00:00 2001
From: Chris Packham <judge.packham@gmail.com>
Date: Wed, 1 Aug 2018 12:19:26 +0530
Subject: [PATCH 0858/1200] spi: kirkwood: add orion-spi compatible string

This matches the compatible string used by the Linux kernel. This will
allow u-boot to use the same device tree files.

Change-Id: I58a42f5220d4aebce4afbc702f14fd942af29305
Signed-off-by: Chris Packham <judge.packham@gmail.com>
Reviewed-by: Jagan Teki <jagan@openedev.com>
[jagan: use armada instead of orion on .data]
Signed-off-by: Jagan Teki <jagan@amarulasolutions.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/59612
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Igal Liberman <igall@marvell.com>
---
 drivers/spi/kirkwood_spi.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/drivers/spi/kirkwood_spi.c b/drivers/spi/kirkwood_spi.c
index 0b5e78d763..210ff86381 100644
--- a/drivers/spi/kirkwood_spi.c
+++ b/drivers/spi/kirkwood_spi.c
@@ -527,6 +527,10 @@ static const struct dm_spi_ops mvebu_spi_ops = {
 	 */
 };
 
+static const struct mvebu_spi_dev armada_spi_dev_data = {
+	.is_errata_50mhz_ac = false,
+};
+
 static const struct mvebu_spi_dev armada_xp_spi_dev_data = {
 	.is_errata_50mhz_ac = false,
 };
@@ -540,6 +544,10 @@ static const struct mvebu_spi_dev armada_380_spi_dev_data = {
 };
 
 static const struct udevice_id mvebu_spi_ids[] = {
+	{
+		.compatible = "marvell,orion-spi",
+		.data = (ulong)&armada_spi_dev_data,
+	},
 	{
 		.compatible = "marvell,armada-375-spi",
 		.data = (ulong)&armada_375_spi_dev_data
-- 
2.22.0

